src/structure.ts

```diff
@@ -338,9 +359,10 @@ interface StructureStorage extends OwnedStructure<STRUCTURE_STORAGE> {
     /**
      * An object with the storage contents.
      */
-    store: StoreDefinition;
+    store: GenericStore;
```

Comment:
This can be better typed:
`store: StoreDefinition;`
It is the same for terminal, container, creep and power creep, which should all use StoreDefinition instead of GenericStore, because it is better typed.